packages/icons/src/Icon.js

```diff
@@ -10,29 +10,33 @@ const Icon = ({ name, title, desc, titleId, descId, ...props }) => {
   ariaLabelledBy += desc && descId ? ` ${descId}` : ''
   ariaLabelledBy = ariaLabelledBy ? ariaLabelledBy : undefined
 
-  return (
-    Component && (
+  if (!Component) {
+    if (process.env.NODE_ENV !== 'production') {
+      console.trace(`icon ${iconName} does not exist`)
+    }
+    return null
+  } else {
+    return (
       <Component
         title={title}
         desc={desc}
         titleId={titleId}
         descId={descId}
-        aria-hidden={!!ariaLabelledBy}
+        aria-hidden={Boolean(ariaLabelledBy)}
```

Comment:
Actually, this logic seems backwards to me.  Shouldn't it be `aria-hidden={!ariaLabelledBy}?`  If there is a label, hidden should be false, because we want the user to find the icon and have its label read by the screen reader.  Otherwise, if there is no label, do hide it as there is nothing for the screen reader to say